channels/drdynvc/tsmf/ffmpeg/tsmf_ffmpeg.c

```diff
@@ -35,12 +35,22 @@
 #define AVMEDIA_TYPE_AUDIO 1
 #endif
 
+#if LIBAVCODEC_VERSION_MAJOR < 54
+#define MAX_AUDIO_FRAME_SIZE AVCODEC_MAX_AUDIO_FRAME_SIZE
+#else
+#define MAX_AUDIO_FRAME_SIZE 192000
```

Comment:
where this sample rate come from?

Reply:
This is defined in the linked PR, nothing I added.

But it seems to be a generic hard coded limit for whatever reason and represents "1 second of 48khz 32-bit audio"

https://www.ffmpeg.org/doxygen/3.2/libavformat_2dvenc_8c_source.html#l00045
https://lists.ffmpeg.org/pipermail/ffmpeg-cvslog/2012-August/053785.html